packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -996,6 +1048,16 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             {
                 command: CommonCommands.SELECT_COLOR_THEME.id,
                 keybinding: 'ctrlcmd+k ctrlcmd+t'
+            },
+            {
+                command: CommonCommands.PIN_TAB.id,
+                keybinding: 'ctrlcmd+k shift+enter',
+                when: '!activeWidgetIsPinned'
+            },
+            {
+                command: CommonCommands.UNPIN_TAB.id,
+                keybinding: 'ctrlcmd+k shift+enter',
```

Comment:
I noticed that pinning does work using the specified keyboard shortcut. However, unpinning does not work for some reason.

Reply:
It turns out this was also a consequence of the 'editor' vs. 'widget' question. An earlier version of the PR had either had two context keys or only `activeWidgetIsPinned` and it was switched to `editor` to agree with VSCode, but the keybindings still referred to `widget`.